open_academy/models/open_academy_session.py

```diff
@@ -1,14 +1,53 @@
-from odoo import models, fields
+from odoo import models, fields, api
+from odoo.exceptions import ValidationError
 
 
 class OpenAcademySession(models.Model):
     _name = "open.academy.session"
     _description = "model of session on open academy"
 
     name = fields.Char(required=True, size=50)
-    duration = fields.Text()
-    start_date = fields.Datetime()
-    number_seats = fields.Integer()
+    duration = fields.Integer()
+    active = fields.Boolean(default=True)
+    start_date = fields.Datetime(default=fields.Date.context_today)
+    end_date = fields.Datetime()
+    number_seats = fields.Integer(required=True, default=1)
+    number_seats_taken = fields.Integer()
     instructor_id = fields.Many2one("res.partner", "instructor")
     course_id = fields.Many2one("open.academy.course", required=True)
     attendees_ids = fields.Many2many("res.partner", "session_res_partner_rel")
+    available_seats = fields.Integer(compute="_compute_percentage_available_seats")
```

Comment:
Missing https://github.com/axlongines/practice/pull/9#discussion_r940387241 This field must not exists.